### ISSUE TITLE
vertex dragging with shift deselects vertex first

### ISSUE THREAD

**Issue by raptor** [bitfighter/bitfighter#505]
_From [watusim...@bitfighter.org](https://code.google.com/u/105427273526970468779/) on April 29, 2015 12:12:11_

Editor: vertex dragging with shift deselects vertex first

_Original issue: http://code.google.com/p/bitfighter/issues/detail?id=505_

**Comment by raptor** [comment]
_From [buckyballreaction](https://code.google.com/u/buckyballreaction/) on April 29, 2015 18:50:53_

I've spent time on this (twice!) and I do not see an easy way to do this.  The problem is that the SHIFT+CLICK is done before the onMouseDragged() event.  Being two separate events makes it difficult to tie them together for this specific case.  

We'd have to figure out how to know that the user was going to drag the items in the SHIFT+CLICK event, which I'm not sure is possible.

Closing as WontFix since the simple solution is to just let go of SHIFT.  This can be reopened if anyone has better ideas.

**Status:** WontFix  
  **Labels:** -019x
